components/src/core/dropdown-toolbar/dropdown-toolbar.component.scss

```diff
@@ -0,0 +1,48 @@
+[dir='rtl'] .pxb-dropdown-toolbar {
+    .pxb-dropdown-toolbar-icon-wrapper {
+        margin-right: -8px;
+        margin-left: 16px;
+    }
+    .pxb-dropdown-toolbar-text-content-container {
+        padding-right: 0;
+    }
+}
+
+.pxb-dropdown-toolbar-icon-wrapper {
+    display: flex;
+    align-items: center;
+    z-index: 1;
+    margin-right: 24px;
+    margin-left: -8px;
+    &.pxb-dropdown-toolbar-no-icon {
+        margin-right: 8px;
+    }
+}
+
+.pxb-dropdown-toolbar-text-content-container {
+    display: block;
+    padding-right: 16px;
```

Comment:
We might be able to leave this padding-right out...the toolbar will already have right padding, so if there are no icons, the right padding would be 32. Might be preferred to leave it up to the user if they need padding on this.